src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -155,99 +130,153 @@ public final void onCompleted(final Run run, @Nonnull final TaskListener listene
-            for (int i = 0; i < tagArr.size(); i++) {
-                statsdTags[i] = tagArr.getString(i);
+            String[] statsdTags = new String[tags.size()];
+            for (int i = 0; i < tags.size(); i++) {
+                statsdTags[i] = tags.getString(i);
             }
 
-            // Report to StatsDClient
-            if (DatadogUtilities.isValidDaemon(getDescriptor().getDaemonHost())) {
-                logger.fine(String.format("Sending 'completed' counter to %s ", getDescriptor().getDaemonHost()));
-                StatsDClient statsd = null;
-                try {
-                    // The client is a threadpool so instead of creating a new instance of the pool
-                    // we lease the exiting one registerd with Jenkins.
-                    statsd = getDescriptor().leaseClient();
-                    statsd.incrementCounter("completed", statsdTags);
-                    logger.fine(String.format("Attempted to send 'completed' counter with tags: %s", Arrays.toString(statsdTags)));
-
-                    logger.fine("Computing KPI metrics");
-                    // Send KPIs
-                    if (run.getResult() == Result.SUCCESS) {
-                        long mttr = getMeanTimeToRecovery(run);
-                        long cycleTime = getCycleTime(run);
-                        long leadTime = run.getDuration() + mttr;
-
-                        statsd.gauge("leadtime", leadTime / THOUSAND_DOUBLE, statsdTags);
-                        if (cycleTime > 0) {
-                            statsd.gauge("cycletime", cycleTime / THOUSAND_DOUBLE, statsdTags);
-                        }
-                        if (mttr > 0) {
-                            statsd.gauge("mttr", mttr / THOUSAND_DOUBLE, statsdTags);
-                        }
-                    } else {
-                        long feedbackTime = run.getDuration();
-                        long mtbf = getMeanTimeBetweenFailure(run);
-
-                        statsd.gauge("feedbacktime", feedbackTime / THOUSAND_DOUBLE, statsdTags);
-
-                        if (mtbf > 0) {
-                            statsd.gauge("mtbf", mtbf / THOUSAND_DOUBLE, statsdTags);
-                        }
+            logger.fine(String.format("Sending 'completed' counter to %s ", getDescriptor().getDaemonHost()));
+            StatsDClient statsd = null;
+            try {
+                // The client is a thread pool so instead of creating a new instance of the pool
+                // we lease the exiting one registered with Jenkins.
+                statsd = getDescriptor().leaseStatDClient();
+                statsd.incrementCounter("completed", statsdTags);
+                logger.fine(String.format("Attempted to send 'completed' counter with tags: %s",
+                        Arrays.toString(statsdTags)));
+
+                logger.fine("Computing KPI metrics");
+                // Send KPIs
+                double THOUSAND_DOUBLE = 1000.0;
```

Comment:
Using a constant named like that seems useless. Let's just use the `1000.0` value directly everywhere it is needed